src/zm_packetqueue.cpp

```diff
@@ -314,7 +314,7 @@ void PacketQueue::clearPackets(const std::shared_ptr<ZMPacket> &add_packet) {
   } // end while
 
   if (keyframe_interval > pre_event_video_packet_count) {
-    Warning("Max Image Buffer setting is too low! Needs to be great than keyframe interval. Keyframe interval is at least %d", keyframe_interval);
```

Comment:
This isn't correct.  The message is correct, I suspect the use of pre_event_video_packet_count is wrong.  Please remove this and I will take a look and correct this properly.  The intention here is to detect and warn that the keyframe interval being sent from the camera is greater than the max queue size, so it is impossible for ZM to queue enough packets.  It is new code I added this just a few days ago.

Reply:
np. I wasn't sure, but if I increased max queue size from 0 (to 120), I was getting other errors. Keyframe interval was around 60 as I remember it.